KubeArmor/go.mod

```diff
@@ -22,104 +22,105 @@ replace (
 )
 
 require (
-	github.com/Masterminds/sprig/v3 v3.2.2
-	github.com/cilium/cilium v1.13.0
+	github.com/Masterminds/sprig/v3 v3.2.3
+	github.com/cilium/cilium v1.13.2
 	github.com/cilium/ebpf v0.10.0
-	github.com/containerd/containerd v1.6.8
-	github.com/containerd/typeurl v1.0.2
-	github.com/docker/docker v20.10.21+incompatible
+	github.com/containerd/containerd v1.7.1
+	github.com/containerd/typeurl/v2 v2.1.1
+	github.com/docker/docker v23.0.6+incompatible
 	github.com/google/uuid v1.3.0
-	github.com/kubearmor/KubeArmor/deployments v0.0.0-00010101000000-000000000000
-	github.com/kubearmor/KubeArmor/pkg/KubeArmorController v0.0.0-20230102134750-3c9ddc923c3f
-	github.com/kubearmor/KubeArmor/protobuf v0.0.0-20220908103453-7b92e248beb9
-	github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
-	github.com/spf13/viper v1.14.0
-	go.uber.org/zap v1.23.0
-	golang.org/x/sys v0.3.0
-	google.golang.org/grpc v1.51.0
-	k8s.io/api v0.26.0
-	k8s.io/apimachinery v0.26.0
-	k8s.io/client-go v0.26.0
-	k8s.io/cri-api v0.25.0
-	k8s.io/utils v0.0.0-20221107191617-1a15be271d1d
-	sigs.k8s.io/controller-runtime v0.13.1-0.20221201045826-d9912251cd81
+	github.com/kubearmor/KubeArmor/deployments v0.0.0-20230510133055-4e30a28b6352
+	github.com/kubearmor/KubeArmor/pkg/KubeArmorController v0.0.0-20230510133055-4e30a28b6352
+	github.com/kubearmor/KubeArmor/protobuf v0.0.0-20230510133055-4e30a28b6352
+	github.com/opencontainers/runtime-spec v1.1.0-rc.2
+	github.com/spf13/viper v1.15.0
+	go.uber.org/zap v1.24.0
+	golang.org/x/sys v0.8.0
+	google.golang.org/grpc v1.55.0
+	k8s.io/api v0.26.4
+	k8s.io/apimachinery v0.26.4
+	k8s.io/client-go v0.26.4
+	k8s.io/cri-api v0.27.1
```

Comment:
Note :- @DelusionalOptimist here `cri-api` is at `v0.27.1`  because `containerd v1.7.1` requires this version, so downgrading `cri-api` to `v0.26.4` will downgrade containerd to `v1.7.0` too